test/parallel/test-repl-deprecations.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+test();
+
+function test() {
+  testParseREPLKeyword();
+}
+
+function testParseREPLKeyword() {
+  const server = repl.start({ prompt: '> ' });
+  const warn = 'REPLServer.parseREPLKeyword() is deprecated';
+
+  common.expectWarning('DeprecationWarning', warn);
+  assert.ok(server.parseREPLKeyword('clear'));
+  common.expectWarning('DeprecationWarning', warn);
```

Comment:
There should be only one call to `common.expectWarning()`. The deprecation warning is only going to be emitted once. What calling `common.expectWarning()` twice does is set up two identical listeners for the `process.on('warning')` event that is only emitted once.